Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -78,6 +78,7 @@ public class JavaScriptLifter: ComponentBase, Lifter {
         }
         
         w.emitBlock(prefix)
+        w.increaseIndentionLevel()
```

Comment:
IIUC, this assumes that the prefix and suffix wrap the generated script into a block (e.g. the function `main`)? I'd prefer not to assume that since it's not required (and probably shouldn't be done by the Duktape and Jerryscript profiles since they don't need it)... If we really care about nicely formatted JS, then maybe we can add a `initialScriptIndentionLevel` field to the profiles?

Reply:
currently all of profiles wrap it in main, but yeah probably no need to complicate it